desktop/dashboard.md

```diff
@@ -125,10 +108,99 @@ Click on a specific container for detailed information about the container. The
 
 You can also use the quick action buttons on the top bar to perform common actions such as opening a CLI to run commands in a container, and perform lifecycle operations such as stop, start, restart, or delete your container.
 
-Click **Port** to open the port exposed by the container in a browser.
+## Explore your images
+
+The **Images**  view is a simple interface that lets you manage Docker images without having to use the CLI. By default, it displays a list of all Docker images on your local disk. To view images in remote repositories, click **Sign in** and connect to Docker Hub. This allows you to collaborate with your team and manage your images directly through Docker Desktop.
+
+The Images view allows you to perform core operations such as running an image as a container, pulling the latest version of an image from Docker Hub, pushing the image to Docker Hub, and inspecting images.
+
+In addition, the Images view displays metadata about the image such as the tag, image ID, date when the image was created, and the size of the image. It also displays **In Use** tags next to images used by running containers. This allows you to review the list of images and use the **Clean up images** option to remove any unused and dangling images from the disk to reclaim space. The Images view also allows you to search images on your local disk and sort them using various options.
```

Comment:
"In Use" also applies to stopped containers. It's only when the container is permanently deleted that the In Use goes away.

Reply:
I suggest removing "and dangling" at this stage and only introducing it later. It's weird terminology and a bit distracting at this point. (Or if you don't want to say "unused" on its own, "unwanted" would be OK too).

Reply:
I think it's worth pulling search and sort out into a separate paragraph to make it more prominent.